examples/website/src/shared/blocks/peerPageBreak.tsx

```diff
@@ -52,23 +53,37 @@ const PeerPageBreakButton = cssRule({
   fontSize: pxToRem(12)
 })
 
-export interface PageBreackBlockStyleProps {
+export interface PageBreakBlockStyleProps {
   isArticle?: boolean
 }
 export interface PageBreakBlockProps {
   peer?: Peer
   text: string
+  richText: RichTextBlockValue
   linkURL: string
   linkText: string
+  linkTarget?: string
+  styleOption?: string
+  templateOption?: string
+  layoutOption?: string
+  imageID?: string | undefined
+  image?: null
 }
 
 export function PageBreakBlock({
   peer,
   text,
+  richText,
   linkURL,
   linkText,
+  linkTarget,
+  styleOption,
+  layoutOption,
+  templateOption,
+  imageID,
+  image,
```

Comment:
again why both `image` and `imageID`?

Reply:
its from before, i did not updated all examples, corrected

Reply:
changed, GH UI not updated, see https://github.com/bajour/wepublish/blob/f/page-break-extend/examples/website/src/shared/blocks/peerPageBreak.tsx